server.js

```diff
@@ -15,6 +15,8 @@ const io = require('socket.io');
 
 // Load configurations
 // if test env, load example file
+
+
```

Comment:
The extra space here is not needed

Reply:
Okay, @adetola-ralph. I have resolved the conflicts.

Reply:
@Onnassiz Please remove the extra space on line 19